README.md

```diff
@@ -89,3 +92,79 @@ VSCode will build the dockerfile inside of `.devcontainer` for you.  If you open
    1. If you are using a `ros2.repos` file, import the contents `Terminal->Run Task..->import from workspace file`
 2. Install dependencies `Terminal->Run Task..->install dependencies`
 3. Develop!
+
+
+### Enable GPU acceleration
+
+Error messages that show lacking GPU acceleration (in docker terminal) 
+
+  $ sudo apt-get update   && sudo apt-get install -y -qq glmark2   && glmark2
```

Comment:
Prefer putting terminal output in a code block

````
```console
foo@bar:~$ whoami
foo
``` 
````

Reply:
Done. Thanks.